microcosm_logging/factories.py

```diff
@@ -113,20 +112,17 @@ def make_json_formatter(graph):
     Used for searchable aggregation.
 
     """
-    fields = {
-        # what was logged?
+    fields = """{
         "loggerName": "%(name)s",
         "levelName": "%(levelname)s",
-        "message": "%(message)s",
```

Comment:
@jessemyers the problem is that `message` is used inconsistently in our code and `%()s` does not properly escape the var `message` properly to ensure the resulting output is json. It will not be json if:

- message is a dictionary. Because then you get an extra set of quotes.
- message is a string with single quotes. They won't be escaped properly and, at least, loggly won't interpret it properly.

If you have another approach for doing this, I'm all ears, but right now the json formatter isn't actually sanitizing inputs to ensure you get json out of it.